index.js

```diff
@@ -150,6 +170,8 @@ class OError extends Error {
  */
 class TaggedError extends OError {}
 
+const DROPPED_TAGS_ERROR = new TaggedError('... dropped tags')
```

Comment:
This will have a misleading stack. How about using a regular object?
```suggestion
const DROPPED_TAGS_ERROR = { name: 'TaggedError', message: '... dropped tags', stack: '' }
```

Reply:
I think this was in large part laziness --- it was hard to get it to typecheck when using a fake object. I'll have another go at it.

Reply:
Was not too bad. Added in 4a508c9.